llvm/test/Transforms/LoopVectorize/single_early_exit_with_outer_loop.ll

```diff
@@ -1,12 +1,11 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
Do these assertions need to be autogenerated here now. Doing so seems to only be dropping checks.